modules/virt-adding-pci-device-hyperconverged-operator.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-configuring-pci-passthrough.adoc
+
+[id="virt-adding-pci-device-hyperconverged-operator_{context}"]
+= Adding PCI devices to the HyperConverged Operator
+To enable the availability of PCI devices in the cluster, add details about the PCI devices to the HyperConverged Operator in the YAML file. The PCI devices are exposed and permitted to be used in the cluster.
+
+.Procedure
+. Use the following command to add the PCI device information to the YAML file.
++
+[source,terminal]
+$ oc get hyperconverged kubevirt-hyperconverged -n openshift-cnv -o yaml
+
++
+.Example
+[source,yaml]
+----
+apiVersion: hco.kubevirt.io/v1
+kind: HyperConverged
+metadata:
+  name: kubevirt-hyperconverged
+  namespace: openshift-cnv
+spec:
+  permittedHostDevices: <1>
+    pciHostDevices: <2>
+      - pciVendorSelector: 10de:1eb8 <3>
+        resourceName: "nvidia.com/TU104GL_Tesla_T4" <4>
+----
+<1> The host devices that are to be used in the cluster.
+<2> PCI devices available on the node.
+<3> The `vendor-ID` and the `device-ID` for the PCI device.
+<4> The name of the hardware device.
+
+.Verification
+* Use the following command to verify that the PCI device was added to the node.
++
+[source,terminal]
+$ oc describe node `cnv-lab.eng.raleigh.redhat.com`
```

Comment:
Same comments here regarding use/run and the `----` around the command

Reply:
Does this actually need the backticks around the node name?

Reply:
Good catch - removed the back ticks.